src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -113,6 +113,7 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
 
         val extendsTypes: List<TypeName>
         val implementsTypes: List<TypeName>
```

Comment:
Is the distinction between these necessary? Seems like a carryover from JavaPoet that has no place here.